### ISSUE TITLE
Specify location of version block

### ISSUE THREAD

**Issue by simonvarey** [Inclusio-Community/json-image-metadata#66]
Version 0.3 of the spec introduces a version block, but it is not yet specified where this block should go in a JIM document.

**Comment by simonvarey** [comment]
https://github.com/Inclusio-Community/json-image-metadata/blob/main/examples/versioned/0.3.2/E001-Vertical_Bar_Chart-0_3_2.svg has `version` at the top level

**Comment by shepazu** [comment]
Yeah, I think that's right, having it at the root. I'm wondering how to spec that out. I don't know if I say elsewhere what the hierarchy is.

**Comment by simonvarey** [comment]
It's not totally explicit, but section 3 has a listing.

**Comment by simonvarey** [comment]
Also, when you say "A `version` block ... contains one or more key-value pairs including one or more of the keys `document` and `jim` and their values" do you mean that a version block can have just `document` or just `jim` but cannot be a non-empty object without either of these keys?

**Comment by simonvarey** [comment]
Actions:
- Specify that the `version` block must be at the top level
- Specify that the `version` block, if present, must contain a valid `jim` property
- Specify that the `document` property is optional, but that if present it has the meaning of the version of the JIM document itself
